src/senaite/core/schema/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+
+from zope.interface import classImplementsFirst
+
+from .fields import IntField
+from .interfaces import IIntField
+
+classImplementsFirst(IntField, IIntField)
```

Comment:
very clever, so we can then be sure that functions for `IIntField` (in this case, `_validate`) have always precedence over others on multiple implementations.